.github/workflows/gradle.yml

```diff
@@ -13,46 +13,48 @@ jobs:
     runs-on: [ubuntu-latest]
     name: Building PA package
     steps:
-    - name: Checkout Performance Analyzer package
-      uses: actions/checkout@v2
-      with:
-        path: ./tmp/pa
-      # fetch the main branch to make it available for spotless ratcheting
-    - name: Fetch 'main' branch
-      working-directory:  ./tmp/pa
-      run: git fetch --depth=1 origin main
-    - name: Set up JDK 1.12
+    - name: Set up JDK
       uses: actions/setup-java@v1
       with:
-        java-version: 1.12
+        java-version: 14
     # dependencies: OpenSearch
     - name: Checkout OpenSearch
       uses: actions/checkout@v2
       with:
         repository: 'opensearch-project/OpenSearch'
         path: OpenSearch
-        ref: '1.0'
+        ref: '1.x'
     - name: Build OpenSearch
       working-directory: ./OpenSearch
-      run: ./gradlew publishToMavenLocal -Dbuild.snapshot=false
+      run: ./gradlew publishToMavenLocal
+
+    # Performance Analyzer in ./tmp/performance-analyzer
+    - name: Checkout Performance Analyzer package
+      uses: actions/checkout@v2
+      with:
+        path: ./tmp/performance-analyzer
       # Explicitly set the docker-compose program path so that our build scripts in RCA can run the program
       # This is necessary because of the Github Actions environment and the workingDir of the Gradle environment
     - name: Set docker-compose path
       run: DOCKER_COMPOSE_LOCATION=$(which docker-compose)
       # Set the vm.max_map_count system property to the minimum required to run OpenSearch
     - name: Set vm.max_map_count
       run: sudo sysctl -w vm.max_map_count=262144
-    - name: Build PA and run Unit Tests
-      working-directory: ./tmp/pa
-      run: ./gradlew build -Dperformance-analyzer-rca.build=true -Dperformance-analyzer-rca.branch=main
+    - name: Build Performance Analyzer and run Unit Tests
+      working-directory: ./tmp/performance-analyzer
+      run: |
+        ./gradlew build -Dperformance-analyzer-rca.build=true \
+          -Dperformance-analyzer-rca.repo="https://github.com/dblock/performance-analyzer-rca.git" \
```

Comment:
Can we change this to the main repo before merging?

Reply:
Sure, but it will break the build.